packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/Informations.tsx

```diff
@@ -10,7 +11,13 @@ const excludedRules = [
 ];
 
 const Informations = (props: WizardStepProps): JSX.Element => (
-  <StepDynamicPublicodes {...props} excludedRules={excludedRules} />
+  <>
+    <Label>
```

Comment:
I would recommend to use <SectionTitle> rather than `Label`

Reply:
Do you miss a word here ?

Reply:
sorry, github hide `<SectionTitle>` when not sorrounded by backtick